 The upstream manpage has compilation instructions which are
 a particularly important section for the pre-package debian
 binary package.

Index: conky/doc/conky.1
===================================================================
--- conky.orig/doc/conky.1	2009-07-08 06:10:34.000000000 +0530
+++ conky/doc/conky.1	2009-07-08 06:13:13.000000000 +0530
@@ -40,35 +40,6 @@
 or stop by #conky on irc.freenode.net if you have questions or want to contribute.
 .PP
 Thanks for your interest in Conky.
-.SH COMPILING
-For users compiling from source on a binary distro, make sure you have the X development
-libraries installed. This should be a package along the lines of "libx11-dev" or
-"xorg-x11-dev" for X11 libs, and similar "\(hydev" format for the other libs required (depending
-on your configure options).
-.PP
-Conky has (for some time) been available in the repositories of most popular distributions.
-Here are some installation instructions for a few:
-.PP
-Gentoo users -- Conky is in Gentoo's Portage... simply use "emerge app-admin/conky" for installation.
-.PP
-Debian, etc. users -- Conky should be in your repositories, and can be installed by doing
-"aptitude install conky".
-.PP
-Example to compile and run Conky with all optional components (note that some configure options may differ for your system):
-.TP 
-\fB\*(T<\fBsh autogen.sh\fR\*(T>\fR \*(T<\fB# Only required if building from the git repo\fR\*(T> 
-.TP 
-\fB\*(T<\fB\&./configure \fR\*(T>\fR\*(T<\fB\-\-prefix=/usr \-\-mandir=/usr/share/man \-\-infodir=/usr/share/info \-\-datadir=/usr/share \-\-sysconfdir=/etc \-\-localstatedir=/var/lib \-\-disable\-own\-window \-\-enable\-audacious[=yes|no|legacy] \-\-enable\-bmpx \-\-disable\-hddtemp \-\-disable\-mpd \-\-enable\-xmms2 \-\-disable\-portmon \-\-disable\-network \-\-enable\-debug \-\-disable\-x11 \-\-disable\-double\-buffer \-\-disable\-xdamage \-\-disable\-xft\fR\*(T> 
-.TP 
-\fB\*(T<\fBmake\fR\*(T>\fR 
-.TP 
-\fB\*(T<\fBmake install\fR\*(T>\fR \*(T<\fB# Optional\fR\*(T> 
-.TP 
-\fB\*(T<\fBsrc/conky\fR\*(T>\fR 
-.PP
-Conky has been tested to be compatible with C99 C, however it has not been tested
-with anything other than gcc, and is not guaranteed to work with other compilers.
-.PP
 .SH "YOU SHOULD KNOW"
 Conky is generally very good on resources. That said, the more you try to make Conky
 do, the more resources it is going to consume.
@@ -2304,6 +2275,35 @@
 \(lahttp://www.sourceforge.net/projects/conky\(ra
 .PP
 #conky on irc.freenode.net
+.SH COMPILING
+For users compiling from source on a binary distro, make sure you have the X development
+libraries installed. This should be a package along the lines of "libx11-dev" or
+"xorg-x11-dev" for X11 libs, and similar "\(hydev" format for the other libs required (depending
+on your configure options).
+.PP
+Conky has (for some time) been available in the repositories of most popular distributions.
+Here are some installation instructions for a few:
+.PP
+Gentoo users -- Conky is in Gentoo's Portage... simply use "emerge app-admin/conky" for installation.
+.PP
+Debian, etc. users -- Conky should be in your repositories, and can be installed by doing
+"aptitude install conky".
+.PP
+Example to compile and run Conky with all optional components (note that some configure options may differ for your system):
+.TP
+\fB\*(T<\fBsh autogen.sh\fR\*(T>\fR \*(T<\fB# Only required if building from the git repo\fR\*(T>
+.TP
+\fB\*(T<\fB\&./configure \fR\*(T>\fR\*(T<\fB\-\-prefix=/usr \-\-mandir=/usr/share/man \-\-infodir=/usr/share/info \-\-datadir=/usr/share \-\-sysconfdir=/etc \-\-localstatedir=/var/lib \-\-disable\-own\-window \-\-enable\-audacious[=yes|no|legacy] \-\-enable\-bmpx \-\-disable\-hddtemp \-\-disable\-mpd \-\-enable\-xmms2 \-\-disable\-portmon \-\-disable\-network \-\-enable\-debug \-\-disable\-x11 \-\-disable\-double\-buffer \-\-disable\-xdamage \-\-disable\-xft\fR\*(T>
+.TP
+\fB\*(T<\fBmake\fR\*(T>\fR
+.TP
+\fB\*(T<\fBmake install\fR\*(T>\fR \*(T<\fB# Optional\fR\*(T>
+.TP
+\fB\*(T<\fBsrc/conky\fR\*(T>\fR
+.PP
+Conky has been tested to be compatible with C99 C, however it has not been tested
+with anything other than gcc, and is not guaranteed to work with other compilers.
+.PP
 .SH COPYING
 Copyright (c) 2005-2009 Brenden Matthews, Philip Kovacs, et. al.
 Any original torsmo code is licensed under the BSD license (see LICENSE.BSD for a copy).
